universum/modules/automation_server/jenkins_server.py

```diff
@@ -29,13 +34,20 @@ def __init__(self, *args, **kwargs):
                                           "is not specified\n\n"
                                           "Please specify the url by using '--jenkins-trigger-url' ('-jtu')\n"
                                           "command-line option or URL environment variable.")
+        self.params = unify_argument_list(self.settings.param_list)
 
-    def trigger_build(self, revision):
-        processed_url = self.settings.trigger_url % revision
+    def trigger_build(self, param_dict=None):
+        processed_url = self.settings.trigger_url
+        if param_dict:
+            self.params.append(urllib.parse.urlencode(param_dict))
+        if self.params:
+            processed_url += '?' + '&'.join(self.params)
 
-        self.out.log("Triggering url %s" % processed_url)
+        self.out.log(f"Triggering url {processed_url}")
         try:
-            requests.get(processed_url)
+            response = requests.post(processed_url)
```

Comment:
It is unusual to use POST http method with query string.
The traditional way is to use query string for GET and payload for POST.

Reply:
https://plugins.jenkins.io/build-token-root/
> Just issue an Http GET or POST
Though I can switch to GET, no problem

Reply:
This is just convention, not a rule